tftpsync/susemanager-tftpsync-recv/configure-tftpsync.sh

```diff
@@ -105,6 +105,10 @@ if [ -z "$PARENT_FQDN" ]; then
         exit 1;
 fi;
 
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+SUMA_IP_DEFAULT=$(join_by " " $( host $SUMA_FQDN | awk '{if($5) print $5; else print $4;}' || echo "" ))
```

Comment:
I think this is too complex. awk has `$NF` to specify number of fields and can be used to print last field as well.

`awk {print $NF}`

And then you do not even need join_by function.

Reply:
Without dollar symbol:
```
awk {print NF}
```

Reply:
Are you sure? Without dollar it will print number of fields, but the the last field. But with `$` it will print the last field.

So this should at the end look:
```bash
SUMA_IP_DEFAULT=$( host $SUMA_FQDN | awk '{print $NF}' || echo "" )
```

Reply:
New version was updated